package.json

```
@@ -12,53 +12,63 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
     "lint:alex": "alex {content,post-templates}/**/*.md",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "lint:md": "markdownlint {content,post-templates}/**/*.md",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@glimmer/component": "^1.0.1",
-    "@glimmer/tracking": "^1.0.1",
+    "@ember/test-helpers": "^2.2.0",
+    "@glimmer/component": "^1.0.3",
+    "@glimmer/tracking": "^1.0.3",
     "alex": "^9.1.0",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "dashify": "^2.0.0",
-    "ember-auto-import": "^1.6.0",
-    "ember-cli": "~3.21.2",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.22.1",
+    "ember-auto-import": "^1.10.1",
+    "ember-cli": "~3.25.2",
+    "ember-cli-app-version": "^4.0.0",
+    "ember-cli-babel": "^7.23.1",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-fastboot": "^2.2.3",
-    "ember-cli-htmlbars": "^5.3.1",
+    "ember-cli-htmlbars": "^5.3.2",
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-netlify": "^0.2.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.0",
-    "ember-data": "~3.21.0",
+    "ember-cli-terser": "^4.0.1",
+    "ember-data": "~3.25.0",
     "ember-export-application-global": "^2.0.1",
-    "ember-fetch": "^8.0.2",
-    "ember-load-initializers": "^2.1.1",
+    "ember-fetch": "^8.0.4",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-metrics": "^1.0.0",
-    "ember-qunit": "^4.6.0",
+    "ember-page-title": "^6.2.1",
+    "ember-qunit": "^5.1.2",
     "ember-resolver": "^8.0.2",
-    "ember-source": "~3.21.1",
-    "ember-template-lint": "^2.11.0",
+    "ember-source": "~3.25.1",
```

I'm not sure yet why Netlify deploy for your pull request failed. I don't think this will make a change, but could you update to the latest `ember-source`, which is 3.25.3?

```suggestion
    "ember-source": "~3.25.3",
```

You will want to commit the new lockfile as well.

---

I found https://github.com/emberjs/data/issues/7361, which indicates that this is due to caching behavior, and it is necessary to wipe away `node_modules` for the build to succeed again. I'm not sure the exact caching behavior that Netlify has/how to address that, though.

Regarding the `ember-source` version, `~3.25.1` actually resolves to `3.25.3` in `package-lock.json`, so this PR is pulling in the latest version regardless.

---

As a project maintainer, would you be able to retry the deploy with a cleared cache as described [here](https://docs.netlify.com/configure-builds/manage-dependencies/#dependency-cache)?

---

Ah. Yep, I can definitely help with deploying after clearing cache.